test/YearnVaultGovernance.test.ts

```diff
@@ -261,16 +259,11 @@ describe("YearnVaultGovernance", function (this: TestContext<
 
             it("denied: random address", async () => {
                 await withSigner(randomAddress(), async (s) => {
-                    await withSigner(randomAddress(), async (s) => {
-                        await expect(
-                            this.subject
-                                .connect(s)
-                                .setYTokenForToken(
-                                    this.weth.address,
-                                    yTokenAddress
-                                )
-                        ).to.be.revertedWith(Exceptions.ADMIN);
-                    });
+                    await expect(
```

Comment:
@MihanixA Need to check why that happened, seems like we have different formatter settings